testing/server.go

```diff
@@ -503,6 +506,22 @@ func (s *DockerServer) statsContainer(w http.ResponseWriter, r *http.Request) {
 	}
 }
 
+func (s *DockerServer) uploadToContainer(w http.ResponseWriter, r *http.Request) {
+	id := mux.Vars(r)["id"]
+	container, _, err := s.findContainer(id)
+	if err != nil {
+		http.Error(w, err.Error(), http.StatusNotFound)
```

Comment:
Could you also please add a test for this case. I know that this test is missing in other handlers, but please add it here x)

Thank you!